.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            // Use IntelliSense to find out which attributes exist for C# debugging
+            // Use hover for the description of the existing attributes
+            // For further information visit https://github.com/dotnet/vscode-csharp/blob/main/debugger-launchjson.md
+            "name": ".NET Core Launch (web)",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            // If you have changed target frameworks, make sure to update the program path.
+            "program": "${workspaceFolder}/src/webapi/bin/Debug/net7.0/webapi.dll",
```

Comment:
This project is using .NET 6, not .NET 7. Originally it was using 7, but we have a policy to avoid it because it is not the "long term support" version of .NET. In other words, we'll be forced to update it next year. But if we use .NET 6, we won't be forced to update until 2025.

Reply:
Also change `webapi.dll` --> `eppeta.webapi.dll`